src/modules/media/queries/get-all-objects-by-collection-id.graphql

```diff
@@ -0,0 +1,81 @@
+query getAllObjectsByCollectionId($collectionId: uuid, $userProfileId: uuid) {
```

Comment:
prefer the prefix "find" so it is more in line with the other queries. Also rename the file

Reply:
Renamed